pom.xml

```diff
@@ -13,34 +13,18 @@
         <maven.compiler.plugin.source>1.8</maven.compiler.plugin.source>
         <maven.compiler.plugin.target>1.8</maven.compiler.plugin.target>
         <testng.version>7.3.0</testng.version>
+        <selenium.version>3.141.59</selenium.version>
+        <bonigarcia.version>3.8.1</bonigarcia.version>
     </properties>
 
     <profiles>
         <profile>
-            <id>Full</id>
-            <activation>
-                <activeByDefault>false</activeByDefault>
-            </activation>
-            <properties>
-                <suite.file>${project.basedir}/src/test/resources/hw1/FullTestSuite.xml</suite.file>
-            </properties>
-        </profile>
-        <profile>
-            <id>AddAndSub</id>
```

Comment:
Why did you delete previous profiles?

Reply:
I've finished making hw3 changes in hw2 branch without branching from master first, so I got confused on how it all will merge and probably I deleted them by mistake. Should I put them back?